src/hooks/global.ts

```diff
@@ -107,15 +101,18 @@ const initGlobal = ({
   options,
   global,
   req,
+  data,
 }: {
   options: TenancyOptions;
   config: Config;
   global: GlobalConfig;
   req: PayloadRequest;
+  data?: Record<string, any>;
```

Comment:
`any` type fails linting. Use `unknown` instead.
```suggestion
  data?: Record<string, unknown>;
```